lib/addons/vpc-cni/index.ts

```diff
@@ -198,7 +292,7 @@ export interface CustomNetworkingConfig {
 
 const defaultProps: CoreAddOnProps = {
   addOnName: 'vpc-cni',
-  version: 'v1.12.6-eksbuild.2',
+  version: 'v1.13.0-eksbuild.1',
```

Comment:
I am a bit concerned why we are using this particular version. 

The latest is 1.13.2. The version that you used is not default for any of the exiting supported versions. v1.12.6-eksbuild.2 is default for 1.27. 
If you intend to use the latest than it should be 1.13.2. However, my recommendation is to use a version which is default for the current highest supported version.

Reply:
@shapirov103 Check this doc - https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html `v1.13.2-eksbuild.1` only version that can be used across quite few versions per the doc. SO lets stay here and update later as needed.

Reply:
Makes sense.